### PR TITLE
Pm 15 tatsat jha neh waiver

### PR DESCRIPTION
**Pull request by TatsatJha**
I changed the waiver to be in the footer, also updated the .toml file to include that disclaimer

### CODE REVIEW

File: layouts/partials/footer.html
Comment (jdamerow) [inline layouts/partials/footer.html:1]: It looks like this file has a slot already planned for the disclaimer text. You just need to uncomment the disclaimerText variable in hugo.toml and then move the printing of the variable to where you currently put the disclaimer directly.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by TatsatJha** [comment]
Requested changes should now be added

**Comment by jdamerow** [comment]
Something is off here:
image

**Comment by TatsatJha** [comment]
I see what's going on, I had the webpage on dark mode, I'll have the styling be dynamic instead

**Pull request closed without merging**